completions/mn

```diff
@@ -0,0 +1,65 @@
+# bash completion for mn                              -*- shell-script -*-
+
+_mn()
+{
+    local cur prev words cword
+    _init_completion || return
+
+    case $prev in
+        --switch)
+             COMPREPLY=($( compgen -W 'default ivs lxbr ovs ovsbr ovsk user
+                 ' -- "$cur" ) )
+             return 0
```

Comment:
Please change all these to simply `return` without a value for consistency with other completions.